styles/styles.css

```diff
@@ -115,7 +115,7 @@ html {
     text-decoration: none;
     display: inline-block;
     font-size: 16px;
-    margin-left: 10%;
+    margin-left: 40%;
```

Comment:
which button is this? Maybe you want to apply a dif CSS class name and move the margin-left to it. button is a generic thing that can be reused.